lib/stink_bomb/configuration.rb

```diff
@@ -6,8 +6,31 @@ def initialize(options = {})
       self.message = options.fetch(:message) { default_message }
     end
 
+    def raise=(error_class)
+      case error_class
+      when true
+        add_bomb(RaiseBomb.new)
+      when false
```

Comment:
Is there a use case where you set it to `true` and then set it to `false` right after?